models/reviews.model.js

```diff
@@ -1,5 +1,33 @@
 const db = require("../db/connection.js")
 
+exports.updateVotesWithReviewID = (req) => {
+
+    if(!req.body.votes) return Promise.reject({status: 400, msg: "400 - Bad input"});
+    const votes = req.body.votes.toString();
+
+    const reviewID = req.params.review_id;
+    if(reviewID.match(/^\d+$/) && votes.match(/^(\d+$|-\d+)$/)){
+        const sql =
+        `
+            UPDATE reviews
+                SET votes = votes + $1
+            WHERE review_id = $2
+            RETURNING *;   
+        `;
+        return db.query(sql, [votes, reviewID])
+        .then((data) => {
+            if(data.rowCount === 0) {
+                return {status: 204, msg: "204 - No content found"};            
+            }else{
+                const reviewObj = { title: data.rows[0].title, votes: data.rows[0].votes }
+                return {status: "Updated successfully", rowCount: data.rowCount, review: reviewObj}        
+            }
```

Comment:
I think I see what you're doing here in trying to organise the response data into an object with info-keys before you send it back. Personally, I view this as taking away from the readability of your code.  My advise would be to keep it simple and just send back the result of the query: 
`return data.rows[0]`

Reply:
You can then send back a simple response from the controller: 
`.then((review) => res.status(200).send({ review: review })`